projects/plugins/jetpack/_inc/client/components/apps-card/index.jsx

```diff
@@ -114,8 +107,4 @@ AppsCard.propTypes = {
 	className: PropTypes.string,
 };
 
-export default connect( state => {
-	return {
-		arePromotionsActive: arePromotionsActive( state ),
-	};
-} )( withRouter( AppsCard ) );
```

Comment:
We should keep wrapping `AppsCard` component with [`withRouter`](https://v5.reactrouter.com/web/api/withRouter) to have access to `location` prop, which is used in the below code line. Otherwise, the event tracked in `trackDownloadClick` is not dispatched due to an error.

https://github.com/Automattic/jetpack/blob/7d9418607207b8e111436395180fa745121ba60a/projects/plugins/jetpack/_inc/client/components/apps-card/index.jsx#L19

Reply:
Thank you for spotting this! I've added the wrapping back in https://github.com/Automattic/jetpack/pull/26659/commits/989598271d182638a937828721d313623e34a1a2.